test/jest-helpers/cli.js

```diff
@@ -12,15 +12,16 @@ expect.extend({toMatchFile});
 
 /**
  * @param {string} args
- * @param {Options} [options=undefined]
+ * @param {Options} [options]
+ * @returns
```

Comment:
Is the type not missing here?

Reply:
With options, the `[]` syntax defaults to undefined.
With the `@returns`, I must've just missed it.